app/components/viewer/ReplayCars.tsx

```diff
@@ -0,0 +1,253 @@
+            );
+
+            const camPos = replay.samples[sampleIndex].velocity.clone().negate().divideScalar(3);
+            camPos.x += 4;
+            camPos.y += 5;
+            velocityRef.current.position.lerp(camPos, 0.4);
+
+            if (timeLineGlobal.followedReplay != null && timeLineGlobal.followedReplay._id === replay._id) {
+                if (orbitControlsRef && orbitControlsRef.current) {
+                    orbitControlsRef.current.target.lerp(replay.samples[sampleIndex].position, 0.4);
+                    if (cameraMode === CameraMode.Cam1) {
+                        const camWorldPos: THREE.Vector3 = new THREE.Vector3();
+                        velocityRef.current.getWorldPosition(camWorldPos);
+                        camera.position.lerp(camWorldPos, 0.3);
+                    }
+                }
+            }
+        }
+    });
+
+    const onPointerLeave = (e: MouseEvent) => {
+        hovered = false;
+    };
+
+    const onPointerMove = (e: MouseEvent) => {
+        hovered = true;
+    };
+
+    const onClick = () => {
+        if (Date.now() - prevOnClick > 100) {
+            if (timeLineGlobal.followedReplay === null
+                || timeLineGlobal.followedReplay._id !== replay._id) {
+                timeLineGlobal.followedReplay = replay;
+            } else {
+                timeLineGlobal.followedReplay = null;
+            }
+            prevOnClick = Date.now();
+        }
+    };
+
+    const [opts, setOpts] = useState({
+        fontSize: 6,
+        color: '#99ccff',
+        maxWidth: 200,
+        lineHeight: 'normal',
+        letterSpacing: 0,
+        textAlign: 'left',
+        materialType: 'MeshPhongMaterial',
+        font: fonts.Comfortaa,
+        anchorX: 'center',
+        anchorY: 'middle',
+        outlineWidth: 0.5,
+        outlineColor: 'black',
+    });
+    const [rotation, setRotation] = useState([0, 0, 0, 0]);
+
+    return (
+        <>
+            <mesh
+                position={[
+                    replay.samples[sampleIndex].position.x,
+                    replay.samples[sampleIndex].position.y,
+                    replay.samples[sampleIndex].position.z]}
+                ref={mesh}
+                scale={1}
+            >
+
+                <primitive
+                    object={fbx}
+                    dispose={null}
+                    ref={stadiumCarMesh}
+                    scale={0.01}
+                    onPointerLeave={onPointerLeave}
+                    onPointerMove={onPointerMove}
+                    onClick={onClick}
+                    receiveShadow
+                    castShadow
+                />
+
+                {showInputOverlay
+                    && <InputOverlay sampleRef={CurrentSampleRef} camera={camera} />}
+
+                <mesh
+                    ref={meshTxt}
+                >
+                    <text
+                        position-z={200}
+                        {...opts}
```

Comment:
I guess this is probably what the linter was complaining about, cause you're spreading the `opts` object.
If so, I'd ignore the rule for this line specifically (not the entire file) - it's totally fair to spread it here.